vspace/src/main/java/edu/asu/diging/vspace/referenceExpose/CitationStyleDefault.java

```diff
@@ -0,0 +1,7 @@
+package edu.asu.diging.vspace.referenceExpose;
+
+public class CitationStyleDefault implements ICitationStyle{
+    
+    String citationStyleName = "defaultZoteroCitation";
```

Comment:
what is this for?

Reply:
?

Reply:
it was not being used anywhere else, not even for the references directly or in directly so I deleted that.